packages/react-dom/src/events/__tests__/ChangeEventPlugin-test.js

```diff
@@ -180,18 +168,19 @@ describe('ChangeEventPlugin', () => {
       expect(e.type).toBe('change');
     }
 
-    if (!ChangeEventPlugin._isInputEventSupported) {
-      return;
-    }
+    // Do not understand this method. Please advise
+    // if (!ChangeEventPlugin._isInputEventSupported) {
+    //   return;
+    // }
```

Comment:
Essentially if the input event isn't supported this test won't test anything

Reply:
how do you check if an input is valid?

Reply:
I think this test as mutated a bit over time. This guard should be fine to just remove at this point. `onInput` will always be supported in the unit test environment so this wouldn't ever get hit.

Reply:
Okay then I'll clean up my code and push. Should be good to go at this point.